gdcdatamodel/models/qcreport.py

```diff
@@ -0,0 +1,116 @@
+from sqlalchemy import Boolean, Integer, Column, DateTime, String, Enum, ForeignKey, text
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship, validates
+
+
+Base = declarative_base()
+SEVERITY = Enum("FAILED", "WARNING", "PASSED", name="error_severity")
```

Comment:
my initial idea was to map this to whatever `bio_submitter_qc` has and that's `WARNING` and `FATAL`, but people didn't like `FATAL` and I think we decided to go with `CRITICAL` instead.